resources/profiles/Geeetech/process/0.06mm Fine @Geeetech A20 0.2 nozzle.json

```diff
@@ -0,0 +1,11 @@
+{
+    "type": "process",
+    "name": "0.06mm Fine @Geeetech A20 0.2 nozzle",
+    "inherits": "fdm_process_geeetech_0.06_nozzle_0.2",
+    "from": "system",
+    "setting_id": "GP_Geeetech A20",
```

Comment:
the format of process setting id is GP + GEEETECH + idx. For instance,  this setting id should be GP_GEEETECH_000. next should be GP_GEEETECH_001. the setting ids should not repeat

Reply:
The format of process setting id has been revised as requested